deploy-build/deploy.sh

```diff
@@ -8,7 +8,7 @@ cd homebrew-appsody
 git checkout -b test${TRAVIS_TAG}${TRAVIS_BUILD_NUMBER}
 cp ../../package/appsody.rb .
 git add appsody.rb
-git commit -m "Travis build: $TRAVIS_TAG$TRAVIS_BUILD_NUMBER" --author="Kyle G. Christianson <christik@us.ibm.com>"
+git commit -m "Travis build: $TRAVIS_TAG$TRAVIS_BUILD_NUMBER" --author="Appsody Devops <appsodydev@gmail.com>"
```

Comment:
This "user" will need to sign the Appsody CLA. Is there a github account associated with this email?

Reply:
Yes good catch! @neeraj-laad should have access to this account so sign the CLA